libs/types/src/result.ts

```diff
@@ -49,6 +49,21 @@ class Ok<T> {
     throw this.value;
   }
 
+  /**
+   * Returns the contained value.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  unsafeExpect(message: string): T {
+    return this.value;
+  }
+
+  /**
+   * Throws the given error message.
+   */
+  unsafeExpectErr(message: string): never {
+    throw new Error(message);
+  }
```

Comment:
Why would you want to use these instead of `unsafeUnwrap` and `unsafeUnwrapErr`?

Reply:
They act more like `assert` with an error message instead of just throwing a value that is potentially less descriptive. The Rust `Result` has `unwrap` and `expect`, and the convention is that `unwrap` is really only for quick and dirty code while you're working on it. If you would use an `assert` in other circumstances, then `expect` is better. Otherwise you should actually handle the error properly.

Reply:
Ok, gotcha. I can see the use of that. Reading the code that uses them, I was pretty confused. Expect makes me think about tests. I think we might want to work on the names a bit.

Maybe `const width = widthResult.assertOk('width is valid');` would mesh more with how I generally think about things in TS.

Reply:
https://github.com/votingworks/vxsuite/pull/2847